backend/btrixcloud/crawlconfigs.py

```diff
@@ -1765,6 +1770,7 @@ async def get_crawl_configs(
             description=description,
             tags=tags,
             tag_match=tag_match,
+            dedupe_coll_id=dedupeCollId,
```

Comment:
```suggestion
            dedupe_coll_id=dedupe_coll_id,
```
To match suggestion above